checkov/main.py

```diff
@@ -62,6 +63,10 @@ def run(banner=checkov_banner, argv=sys.argv[1:]):
     add_parser_args(parser)
     config = parser.parse_args(argv)
 
+    if config.add_check:
+        prompt.Prompt().go()
```

Comment:
maybe I'm missing something, but why use the `go()` method and the logic inside and not just do it here like that
```python
resp = prompt.Prompt()
check = prompt.Check(resp.responses)
check.action()
```

Reply:
I agree this is better. Maybe _I'm_ missing something, but I ended up needing to explicitly create an `action()` and changing the prompt variable to `chosen_action` to get this working. I'm happy for any additional suggestions: 
```python
    def action(self):
        # Call the user-selected action (add, remove, modify - leaves room for more actions)
        getattr(self, self.chosen_action)()
```

Reply:
that's fine 🙂 maybe it is possible with the `@property` decorator to make it work, but it is totally fine, how it is right now 👍